src/components/Reports/GDPR/SubscriberGdpr.styles.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+import fonts from '../../styles/fonts';
+
+export const TextColumn = styled.div`
+  p {
+    font-size: ${fonts.fontLvl3};
```

Comment:
looks great, i would just add a line-height: 18px; so it remains like the rest

Reply:
it's has now a line-height with that value